docs/pages/style-spec.js

```diff
@@ -1053,14 +1055,51 @@ export default class extends React.Component {
                                             }
                                         }`)}
                                 </div>
-                                Then the style could refer to this sprite image
-                                by creating a symbol layer with the layout
-                                property
-                                <code>"icon-image": "poi"</code>, or with the
-                                tokenized value{' '}
-                                <code>"icon-image": "{`{icon}`}"</code> and
-                                vector tile features with a <code>icon</code>{' '}
-                                property with the value <code>poi</code>.
+                                <p>
+                                    Then the style could refer to this sprite
+                                    image by creating a symbol layer with the
+                                    layout property
+                                    <code>"icon-image": "poi"</code>, or with
+                                    the tokenized value{' '}
+                                    <code>"icon-image": "{`{icon}`}"</code> and
+                                    vector tile features with a{' '}
+                                    <code>icon</code> property with the value{' '}
+                                    <code>poi</code>.
+                                </p>
+                                <p>
+                                    Apart from the required <code>width</code>,
```

Comment:
This is minor, but maybe another space here?
```suggestion
                                    Apart from the required <code>width</code>,{' '}
```